app/containers/UserCalendar/index.js

```diff
@@ -34,21 +36,25 @@ const UserCalendar = ({ loading, viewer, user, router, refetch }) => {
       <AppContent>
         <Content user={selectedUser} />
       </AppContent>
-      <AppMenu>
-        <AppMenu.Context.Consumer>
-          {({ target }) => (
-            <TeamMembers
-              scrollTarget={target}
-              setUser={({ id }) =>
-                id === viewer.user.id
-                  ? Router.push(...paths.me.calendar())
-                  : Router.push(...paths.users.calendar({ id }))
-              }
-              selected={selectedUser}
-            />
-          )}
-        </AppMenu.Context.Consumer>
-      </AppMenu>
+      {['Admin', 'Member'].includes(viewers.user.role) && (
```

Comment:
Use [AuthHelper](https://github.com/coding-blocks/gondor/blob/master/app/utils/authHelper.js) here.

Also if you remove AppMenu, you also need to not wrap the Content with AppContent.